### DISCUSSION TITLE
USD file sequences

### DISCUSSION THREAD

**Discussion by yolov** [Autodesk/maya-usd#1371] [Ideas]
Just wondering if exporting/reading USD file sequences is being considered?
If there's baked animation that can make the file huge, I guess it would be better if it was broken to one frame per file.

Alex

**Comment by sepux** [comment]
Bringing back this question, wondering if there is any way to export USD file sequences yet in USD .16 or .17 - thanks.

**Comment by santosd** [comment]
This is still not something that has been implemented at this time. We could however, convert this to an issue and see if anyone in the community would like to work on implementing this. I will also bring it up to the team and see if this is something that fits into our current scope.
